modules/acquia_cms_common/config/optional/filter.format.filtered_html.yml

```diff
@@ -20,17 +20,16 @@ filters:
     settings: {  }
   filter_url:
     id: filter_url
-    provider: filterclear
+    provider: filter
     status: true
-    weight: -47
+    weight: -46
     settings:
       filter_url_length: 72
   filter_html:
     id: filter_html
     provider: filter
     status: true
-    weight: -49
-    # Manually added the <p> tags for alignment buttons per https://www.drupal.org/project/drupal/issues/2649546.
```

Comment:
@panshulK Add back this commented line. So, that we would know why those classes were added in <p> tag. 
:)